src/main/java/com/endava/internship/warmup/domain/service/ArrayProcessorWithForLoops.java

```diff
@@ -111,7 +181,44 @@ public int findSecondMax(final int[] input) {
      */
     @Override
     public int[] mergeSortedArrays(int[] input, int[] input2) throws IllegalArgumentException {
-        throw new UnsupportedOperationException("Method not implemented yet");
+        final int N = input.length;
```

Comment:
give more descriptive names for variables/constants. Constants must be declared as final and static, and their names must be only with upper case.